### PR TITLE
Merged CL:100416 into CL:0002324

### LINKED ISSUE #21: merge CL:1000416 'myoepithelial cell of lactiferous gland' into CL:0002324 'myoepithelial cell of mammary gland'
```
obsolete CL:1000416

move equiv axiom to  CL:0002324 

retain 'mammary' in primary label. Use 'lactiferous' in exact syn, attribute to 
FMA
```

Original issue reported on code.google.com by `cmung...@gmail.com` on 20 Feb 2013 at 6:21

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #21 - picked this up cause it looked like a simple fix

UBERON:0006954 uses CL:1000416 -> needs to updated to CL:0002324 in UBERON

@nicolevasilevsky @matentzn tried using

`sh run.sh owltools --use-catalog cl-edit.owl --obsolete-replace CL:1000416 CL:0002324 -o -f owl cl-edit.owl
`
(not sure what the owl after -f was so tried both owl and obo)

Created insane diff - basically everything got replaced, so decided to do this manually instead - not sure if it needs looking into

@addiehl @cmungall would like a 2nd eye on this to make sure it is done right - I did this mechanically and don't really know the science and all.

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -17612,10 +17612,14 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true"^^xsd:string) obo:CL_0002323 ob
...
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002324 "CALOHA:TS-2378"^^xsd:string)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002324 "FMA:67800"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "FMA:67800"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_0002324 "basal cell of lactiferous duct"^^xsd:string)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0002324 "myoepithelial cell of lactiferous gland")
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0002324 "cell"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0002324 "myoepithelial cell of mammary gland"^^xsd:string)
-SubClassOf(obo:CL_0002324 obo:CL_0000185)
-SubClassOf(obo:CL_0002324 obo:CL_0002327)
+EquivalentClasses(obo:CL_0002324 ObjectIntersectionOf(obo:CL_0000185 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001911)))
+SubClassOf(Annotation(oboInOwl:is_inferred "true"^^xsd:string) obo:CL_0002324 obo:CL_0000185)
```
Comment (cmungall) [inline src/ontology/cl-edit.owl:17622]: odd to assert this, but this may be legacy practice in CL

**Review by cmungall — APPROVED** [review]

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (shawntanzk) [reply src/ontology/cl-edit.owl]: yeah didn't notice that, don't know how I feel about that so I've removed it, thanks for the catch

**Comment by shawntanzk** [comment]
Had to remove axioms brought in my uberon import that was causing issues as it was bringing it the obsolete terms and axioms still

**Comment by shawntanzk** [comment]
Current issues are with uberons use of the obsolete terms, will fix here https://github.com/obophenotype/uberon/issues/2188 and for now will fix by changing it in CL's uberon import too

**Pull request merged**